tests/integration/scrapers/scrapers-new-test.js

```diff
@@ -0,0 +1,36 @@
+const imports = require('esm')(module);
+const { join } = require('path');
+const test = require('tape');
+const exec = require('child_process').execSync;
+
+const shared = join(process.cwd(), 'src', 'shared');
+const lib = join(shared, 'lib');
+
+const fs = imports(join(lib, 'fs.js'));
+const schema = imports(join(lib, 'schema.js'));
+const runScraper = imports('./run-scraper.js').default;
+
+const command = 'git diff --name-only HEAD';
```

Comment:
We use different diffing strategies depending whether we are running for a PR (diff against master) or just a regular commit (diff against previous commit). Would recommend keeping the env variable `FILE_MODIFIED` which we set in the `package.json` scripts

Reply:
I fail to see how the result differs between running it via the npm script and running it within the test, can you elucidate?

Reply:
See `.github/workflows/test.yml`, we run `git diff --name-only origin/master` when running the workflow in a PR as we want to compare against master. This test will need to do the same to avoid a regression.

Reply:
Got it, was not aware.

So in CI we run `git diff --name-only origin/master`, while locally we run `git diff --name-only HEAD`, yes? 

Will memorialize in the test.